frontend/src/components/pages/teacher/DistributeAssessmentPage.tsx

```diff
@@ -1,8 +1,99 @@
-import React from "react";
-import { Text } from "@chakra-ui/react";
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { useState } from "react";
+import { Button, HStack, Spacer, VStack } from "@chakra-ui/react";
+
+import type { BreadcrumbType } from "../../sessions/distribute/FormBreadcrumb";
+import FormBreadcrumb from "../../sessions/distribute/FormBreadcrumb";
+import AddInformation from "../../sessions/distribute/steps/AddInformation";
+import ChooseAssessment from "../../sessions/distribute/steps/ChooseAssessment";
+import ChooseClass from "../../sessions/distribute/steps/ChooseClass";
+import Review from "../../sessions/distribute/steps/Review";
 
 const DistributeAssessmentPage = (): React.ReactElement => {
-  return <Text>Placeholder for Distribute Assessment page.</Text>;
+  const [page, setPage] = useState(0);
+
+  const [testId, setTestId] = useState("");
+  const [classId, setClassId] = useState("");
+  const [startDate, setStartDate] = useState<Date | null>(null);
+  const [endDate, setEndDate] = useState<Date | null>(null);
+  const [notes, setNotes] = useState("");
+
+  const BREADCRUMB_CONFIG: BreadcrumbType[] = [
+    { header: "Choose an assessment", page: 0 },
+    { header: "Choose a classroom", page: 1 },
+    { header: "Add Information", page: 2 },
+    { header: "Review", page: 3 },
+  ];
+
+  const renderPageContent = () => {
```

Comment:
you could probably just render all of these into an array and then use the page # to index into it, but this is okay too